docs/install/windows.md

```diff
@@ -1,37 +1,405 @@
+
+Install Windows Subsystem for Linux:  wsl.exe --update  
+
+`wsl.exe --list --online` shows you your options and 
+`wsl.exe --install Ubuntu-24.04` (or any other) selects an option.
+  
+------------------------------------------------------------ 
+
+### Step 10: (If you are installing NEURON in a Virtual Machine) 
+
+Turn off your VM and at your computer as **administrator** put in PowerShell 7: 
+
+`
+Set-VMProcessor -VMName "NameOfYourVM" -ExposeVirtualizationExtensions $true
+`
+
+For example:
 
-From a powershell with **administrator** priviledges, run:
 ```powershell
-PS C:\workspace\nrn> .\ci\win_download_deps.cmd
+Set-VMProcessor -VMName "Windows 11 dev environment" -ExposeVirtualizationExtensions $true 
+```
+
+ 
+------------------------------------------------------------ 
+
+ 
+### Step 11: We need execute the “mingw64.exe” 
+So, look for "msys64", go to folder “msys64” and execute `mingw64.exe`.
+
+Via folder: open folder `C:\msys64` (probably) and double click in the executable `mingw64` 
+
+Via terminal: go to the location of the execuable `./mingw64.exe` (`C:\msys64` probably or `\\wsl.localhost\Ubuntu-24.04\mnt\c\msys64`) and execute it 
+
+```
+./mingw64.exe
+```
+ or `/mnt/c/msys64$ ./mingw64.exe `
+ 
+It will open a terminal. It is not PowerShell, it is not CMD, it is a mingw64 teminal.
+ 
+------------------------------------------------------------ 
+## How to build NEURON
+
+### Step 12: Creating the exe
+
+Now, inside this terminal (it is a mingw64 terminal, see step above) go to `C:\Users\User\Neuron\nrn\ci` and run 
+
+```
+./win_build_cmake.sh 
+```
+ 
+It will create an exe file `nrn-nightly-AMD64` (at `C:\Users\User\Neuron\nrn\nrn-nightly-AMD64`).
+ 
+------------------------------------------------------------ 
+
+### Step 13: Running the exe
+
+Double click at `nrn-nightly-AMD64` and it will install NEURON. 
+It is a standard installation. 
+
+That means the files created at `C:\Users\User\Neuron\nrn\build\bin` will be coped to:  `C:\nrn\bin` and the "installation" will appear at `C:\nrn` and `C:\nrn-install` 
+ 
+------------------------------------------------------------ 
+
+
+### Step 14: Using and modifying NEURON file... 
+You can replace any new `.dll` file at `C:\nrn\bin` and the NEURON will present the new feature implemented. 
+ 
+Example: 
+
+#### Step 14.A: Checking the `nrngui` 
+Run `nrngui` (inside the folder `C:\Users\User\Desktop\NEURON 9.0.0 AMD64` double click on `nrngui`)
+
+It will show something like:
+
+```
+NEURON -- VERSION X.X.X 2024-07-09
+
+Duke, Yale, and the BlueBrain Project -- Copyright 1984-2022
+See http://neuron.yale.edu/neuron/credits
+
+oc>
+```
+
+Close this NEURON window.
+
+#### Step 14.B: Making some changes at `init.cpp`
+
+Open the `init.cpp` file (at `C:\Users\User\Neuron\nrn\src\nrnoc\init.cpp`  or at `C:\msys64\home\User\Neuron\nrn\src\nrnoc\init.cpp`) look for `hoc_last_init(void)` and the follow code inside it:
```

Comment:
... and add the following code inside it:

Also, personal preference, but I'd put curly braces for every if.